core/lib/dr_events.h

```diff
@@ -641,6 +641,12 @@ typedef struct _dr_fault_fragment_info_t {
      * depending on the type of cache consistency being used by DR.
      */
     bool app_code_consistent;
+    /**
+     * The recreated ilist for this fragment. This also contains metadata
+     * added by various clients, which may be helpful in restoring state
```

Comment:
I think it would be good to add some info to https://dynamorio.org/API_BT.html#sec_translation as well about the ilist being passed to the restore state callback.

Reply:
Done.